### DISCUSSION TITLE
[Suggestion] Make "extend deadline" price scale per day added for current quota

### DISCUSSION THREAD

**Discussion by chibranche** [Malcolm-Q/LC-LateGameUpgrades#461] [Ideas]
Add a new configuration value so that each day added with "extend deadline" cost more and more until quota is reached.

Currently "Extend Deadline Price" is fixed, so each day added cost the same, and it can be difficult to find a value that seems fair and engaging for longer runs.

Suggestion would be to add a new value "Cost increase per day" so that nth day bought for current quota would cost :

cost for nth day = "Extend Deadline Price" + (n-1) * "Cost increase per day"

The variable "n" would reset on run reset or quota reached

This way we would have a scaling cost and higher quota would be more and more difficult to reach. Of course value could be set to 0 to keep current behaviour

**Comment by WhiteSpike** [comment]
I would have to store somewhere the amount of days you got to that point and then calculate the price of the command according to your input and the configuration.
Right now, the logic is all in one place and I can maybe divide this and make it more feasible for price calculation configurations.

**Comment by chibranche** [comment]
I can take a look in the code and test things out, then make a suggestion to make your life easier if you ever want to work on this feature. As we discussed in the discord, biggest hurdle would be that we need a way to efficiently display this to the players (new confirmation UI screen).

I'll test things out and swing by in the discord if I make any progress
